packages/gasket-log/test/client.test.js

```diff
@@ -23,7 +23,7 @@ describe('Log', function () {
 
   it('has logger for each level', function () {
     // had to change from process.stdout "write" to get passing test
-    const write = jest.spyOn(console, 'log').mockImplementation(msg => msg);
+    const write = jest.spyOn(console, 'log').mockImplementation((msg) => msg);
```

Comment:
Are we going to keep `@gasket/log`? If not, we need a ticket to remove it if it's not going to be included in this PR.

Same for removing `@gasket/plugin-log`.